src/main/java/com/codeninjas/inventoryservice/domain/Inventory.java

```diff
@@ -0,0 +1,29 @@
+package com.codeninjas.inventoryservice.domain;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Generated;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(value = "inventory")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Inventory {
+
+    @Id
+    @Generated
+    private String inventoryId;
+
+    @NotNull(message = "item code can't be null")
+    @NotBlank(message = "item code can't be blank")
+    private String itemCode;
+
+    @NotNull(message = "quantity can't be null")
+    @NotBlank(message = "quantity can't be blank")
```

Comment:
Please remove this line
@NotBlank(message = "quantity can't be blank")
this validation will not work for integer

Reply:
I have removed it

Reply:
Please make sure to push the changes.